equinix/resource_fabric_cloud_router_acc_test.go

```diff
@@ -3,103 +3,125 @@ package equinix
 import (
 	"context"
 	"fmt"
-	"testing"
-
-	"github.com/equinix/terraform-provider-equinix/internal/config"
-
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
+	"github.com/equinix/terraform-provider-equinix/internal/config"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"testing"
 )
 
-func TestAccCloudRouterCreate(t *testing.T) {
+func TestAccCloudRouterCreateOnlyRequiredParameters_PFCR(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
 		CheckDestroy: checkCloudRouterDelete,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCloudRouterCreateConfig("fg_tf_acc_test"),
+				Config: testAccCloudRouterCreateOnlyRequiredParameterConfig("fcr_tf_acc_test"),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
-						"equinix_fabric_cloud_router.test", "name", "fg_tf_acc_test"),
+						"equinix_fabric_cloud_router.test", "name", "fcr_tf_acc_test"),
 				),
 				ExpectNonEmptyPlan: false,
 			},
 			{
-				Config: testAccCloudRouterCreateConfig("fg_tf_acc_update"),
+				Config: testAccCloudRouterCreateOnlyRequiredParameterConfig("fcr_tf_acc_update"),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
-						"equinix_fabric_cloud_router.test", "name", "fg_tf_acc_update"),
+						"equinix_fabric_cloud_router.test", "name", "fcr_tf_acc_update"),
```

Comment:
Same as the other comment. We need to check as exhaustive a list of the resource attributes as we can in these tests to verify they're working properly.

Reply:
There are a few names that need to be updated to `_PFCR` suffix.